public/app/features/admin/AdminFeatureTogglesPage.tsx

```diff
@@ -59,7 +59,7 @@ export default function AdminFeatureTogglesPage() {
           {isError && getErrorMessage()}
           {isLoading && 'Fetching feature toggles'}
           {featureMgmtState?.allowEditing && <EditingAlert />}
-          {featureToggles && (
+          {!isLoading && featureToggles && (
```

Comment:
for some reason, this was breaking things with the new k8s api. this should make react more predictable